srcpkgs/openmsx/template

```diff
@@ -1,17 +1,17 @@
 # Template file for 'openmsx'
 pkgname=openmsx
-version=0.13.0
-revision=2
+version=0.14.0
+revision=1
 short_desc="The MSX emulator that aims for perfection"
 maintainer="Juan RP <xtraeme@voidlinux.eu>"
-license="GPL-3"
+license="GPL-2.0"
 homepage="http://openmsx.sourceforge.net"
 distfiles="https://github.com/openMSX/openMSX/releases/download/RELEASE_${version//\./_}/openmsx-${version}.tar.gz"
-checksum=41e37c938be6fc9f90659f8808418133601a85475058725d3e0dccf2902e62cb
+checksum=eb9ae4c8420c30b69e9a05edfa8c606762b7a6bf3e55d36bfb457c2400f6a7b9
 hostmakedepends="pkg-config python"
 makedepends="SDL-devel glew-devel libao-devel libtheora-devel libvorbis-devel
  libpng-devel libxml2-devel SDL_ttf-devel tcl-devel"
-nopie=yes
+only_for_archs="i686 x86_64"
```

Comment:
Did earlier versions of OpenMSX compile on ARM architectures on Void? It does on Debian, at least...

Reply:
Unless there is a very good reason do it the other way round, remove "only_for_arches" and add "broken=reason" for specific platforms.